tests/test_templatetags.py

```diff
@@ -10,36 +12,33 @@ def test_page_meta(self):
         """
         Test page-level templatetags
         """
-        page1, __ = self.get_pages()
```

Comment:
We can redefined get_pages, I think we could remove a lot of our changes to the tests if self.pages could be used.

The aim with a 3rd party codebase is to leave as small a footprint as possible and changing the bare minimum.

We could copy get_pages from the app helper and fix it in the base class in this project: 
- https://github.com/nephila/django-app-helper/blob/461cc3503693b56cef307bbd6cbef4e67349a82a/app_helper/base_test.py#L408
- https://github.com/nephila/djangocms-page-meta/blob/942dc76fcc8947960d546c4e8a21a33e4cf3c468/tests/__init__.py#L13